core/trino-main/src/main/java/io/trino/sql/analyzer/FeaturesConfig.java

```diff
@@ -1103,4 +1106,17 @@ public FeaturesConfig setDisableSetPropertiesSecurityCheckForCreateDdl(boolean d
         this.disableSetPropertiesSecurityCheckForCreateDdl = disableSetPropertiesSecurityCheckForCreateDdl;
         return this;
     }
+
+    @NotNull
```

Comment:
nit: I do not think we annotate with @NotNull typically.

Reply:
We do for validation, see `getJoinMaxBroadcastTableSize`